extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -664,9 +664,38 @@ interface HibernateOrmConfigPersistenceValidation {
 
         /**
          * Enables the Bean Validation integration.
+         *
+         * @deprecated Use {@link #modes()} instead.
          */
+        @Deprecated(since = "3.19", forRemoval = true)
         @WithDefault("true")
         boolean enabled();
+
+        /**
+         * Defines how the Bean Validation integration behaves.
+         */
+        @WithDefault("AUTO")
```

Comment:
Using `auto` as that's what will appear as possible values in the doc.

```suggestion
        @WithDefault("auto")
```